frontend-v2/src/features/simulation/useSimulation.ts

```diff
@@ -70,6 +76,7 @@ export default function useSimulation(
     simulate,
     JSON.stringify(simInputs),
     page,
+    runSimulation,
   ]);
```

Comment:
At some point, I think I should rewrite this hook, to see if I can reduce the number of dependencies for this effect.